pkg/validation/options.go

```diff
@@ -225,6 +225,16 @@ func Validate(o *options.Options) error {
 		})
 	}
 
+	if len(o.TrustedIPs) > 0 && o.ReverseProxy {
+		fmt.Fprintln(os.Stderr, "WARNING: trusting of IPs with --reverse-proxy poses risks if a header spoofing attack is possible.")
```

Comment:
This is good, any reason not to use our logger package though?

Reply:
Isn't it not initialised yet?

Reply:
That is a fantastic point! Ignore me 😅